heroku/deploy-a-django-app.md

```diff
@@ -296,6 +273,33 @@ Use the following baseline to get started:
 }
 ```
 
+## Set up GitHub Actions for CI
+
+For Heroku deployments, we use GitHub Actions instead of Travis for CI (continuous integration). Read the [how-to to set up GitHub Actions](https://github.com/datamade/how-to/blob/master/ci/github-actions.md).
+
+## Provision Heroku resources
```

Comment:
You mentioned in the issue, and I agree, that increasingly, we most often come to these docs with our application already configured. For that reason, I think we should _start_ with provisioning Heroku resources, and move `Set up application code for Heroku` to the second to last section, as an optional step for legacy projects.

Reply:
Changed here: 44fd527. I added a little bit of context to intro text, too.